index.js

```diff
@@ -331,6 +333,10 @@ module.exports = class Hyperswarm extends EventEmitter {
     return peerInfo
   }
 
+  _handleUpdate = () => {
+    this.stats.updates++
+  }
```

Comment:
this can just be a normal _handleUpdate () { ... } function as the EventEmitter forwards `this`

Reply:
ooh neat, I did not know that. That's handy